src/test/integration/backer.t.sol

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.4.23;
+
+import "ds-test/test.sol";
+
+import "./system.t.sol";
+import "../../backer.sol";
+
+contract BackerLike {
+    function file(address usr) public;
+}
+
+contract BackerTest is DSTest, SystemTest {
```

Comment:
all tests in systemTest will be executed with the backer adapter

Reply:
Why not simplelender?

Reply:
system tests with SimplerLender are executed too.